Automattic/woocommerce-payments
# add: test instructions icon animation

Fixes https://github.com/Automattic/woocommerce-payments/issues/9399

#### Changes proposed in this Pull Request









Adding a little animation on hover, click, success of the clickable payment method icon - as discussed here: pfHfG4-gT-p2#comment-238

I needed to change the markup being generated:
- Before: button and test number were two separate elements.
- Now: button wraps the test number, to make the click target bigger.

Demo on blocks and cart checkout:
![2024-08-21 12 30 57](https://github.com/user-attachments/assets/0f4d45c4-2ac4-45b6-826f-313a54caa554)
![2024-08-21 12 30 39](https://github.com/user-attachments/assets/b3d60baf-2b75-4873-a20e-f724670f777b)

Also tested on a bunch of other theme variations (TT4, Storefront, TT3, TT2 - blocks & shortcode checkout):
<img width="483" alt="Screenshot 2024-08-21 at 12 02 17 PM" src="https://github.com/user-attachments/assets/ed7b612c-6f0a-4a68-9013-244384776c73">
<img width="776" alt="Screenshot 2024-08-21 at 12 02 13 PM" src="https://github.com/user-attachments/assets/d0e9fe05-1c40-455c-b8f4-520b1a600f5f">
<img width="479" alt="Screenshot 2024-08-21 at 12 01 54 PM" src="https://github.com/user-attachments/assets/d87362dd-6046-4892-9380-85744c6752a0">
<img width="641" alt="Screenshot 2024-08-21 at 12 01 45 PM" src="https://github.com/user-attachments/assets/5bf9432e-7457-4589-9cc0-61e263501df5">
<img width="535" alt="Screenshot 2024-08-21 at 11 48 38 AM" src="https://github.com/user-attachments/assets/59b41621-b33b-45dd-997f-b5d1b0eed577">
<img width="673" alt="Screenshot 2024-08-21 at 11 46 18 AM" src="https://github.com/user-attachments/assets/f740d5a4-bc56-4bfa-8b92-a669c8801df6">

#### Testing instructions





- Ensure WooPayments is in test mode
- Go to the checkout (you can test both block and shortcode)
- Clicking on the test number should copy the test number
- Tested for card, Becs, and SEPA

-------------------


**Post merge**
